examples/go/hello-lambda/Makefile

```diff
@@ -5,7 +5,7 @@ all: run
 .PHONY: build-linux
 ## build-linux: build the executable for linux
 build-linux:
-	GOOS=linux GOARCH=amd64 go build -o dist/hello-lambda
+	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o dist/hello-lambda
```

Comment:
We need to static link otherwise the AWS runtime fails to find a C library

Reply:
what's the CGO dependency?

Reply:
I don't know, but 
```
$ ldd dist/com.mattermost.apps/server/dist/plugin-linux-amd64
	linux-vdso.so.1 (0x00007ffd43bf6000)
	libpthread.so.0 => /usr/lib/libpthread.so.0 (0x00007fe1cdf0c000)
	libc.so.6 => /usr/lib/libc.so.6 (0x00007fe1cdd3f000)
	/lib64/ld-linux-x86-64.so.2 => /usr/lib64/ld-linux-x86-64.so.2 (0x00007fe1cdf55000)
```

clearly shows that CGO is used _somewhere_. Maybe it's the server?

Reply:
Also, why did it work for me without it (in AWS?)

Reply:
@levb I have no idea